### DISCUSSION TITLE
Calendar time zone setting

### DISCUSSION THREAD

**Discussion by AidenLiu0520** [vkurko/calendar#367] [Q&A]
How components initialise time zones for different time zones

**Comment by Ashk2a** [comment]
Same question, I didn't see the fullcalendar equivalent of "timeZone" option

**Comment by vkurko** [comment]
There is currently no equivalent of `timeZone` option. But even in FullCalendar this option without additional libraries only supports `"local"` and `"UTC"`.

**Comment by vkurko** [comment]
[timeZone](https://github.com/vkurko/calendar?tab=readme-ov-file#timezone) option is available since v5.7.0.
